apps/privacy-panel/style/lists.css

```diff
@@ -269,32 +269,32 @@ ul li p + label:not([for]) {
  * Range inputs
  */
 
-input[type=range] {
+input {
```

Comment:
Please don't generalize all these specific `input[type=range]` rules copied from `apps/settings/style/lists.css`, there is no need to do that.